src/Polysemy/Internal/TH/Effect.hs

```diff
@@ -36,16 +36,16 @@ import Prelude hiding ((<>))
 
 import Control.Monad
 import Data.Bifunctor
-import Data.Char                      (toLower)
+import Data.Char                             (toLower)
```

Comment:
Any chance we can just use one space for these?

Reply:
Ok.